client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -1,76 +1,67 @@
+                                    match="text"
+                                    required
+                                    data-align="start">
+                            <span ng-bind="$item.text"></span>
+                        </oui-select>
+                    </oui-field>
+                    <oui-field label="{{configuration.key}}"
+                               size="s"
+                               ng-if="configuration.type === 'toggle'"
+                               help-text="{{configuration.description}}">
+                        <oui-radio-toggle-group model="configuration.selectedValue.id" required>
+                           <oui-radio ng-repeat="item in configuration.availableValues"
+                                      text="{{item.text}}"
+                                      value="item.id"></oui-radio>
+                        </oui-radio-toggle-group>
+                    </oui-field>
                 </div>
-
-                <form class="form-horizontal">
-                    <div class="form-group mb-md-1" data-ng-repeat="configuration in configurationsCtrl.configurations track by $index">
-                        <label class="control-label col-md-4"
-                               data-ng-attr-for="{{configurationsCtrl.edit.value && (configuration.key + 'Field') || null}}">
-                            <span class="mr-2" data-ng-bind="configuration.key"></span>
-                            <span class="fa fa-info-circle"
-                                  data-uib-tooltip="{{configuration.description}}"
-                                  data-tooltip-append-to-body="true">
-                            </span> :
-                        </label>
-                        <div class="col-md-8">
-                            <span class="d-block form-control-static"
-                                  data-ng-if="!configurationsCtrl.edit.value">
-                                <span data-ng-bind="configuration.value"></span>
-                                <span data-i18n-static="{{'unit_configuration_' + configuration.unit}}" data-ng-if="configuration.unit"></span>
-                            </span>
-                            <div class="oui-select oui-select_inline" data-ng-if="configurationsCtrl.edit.value">
-                                <select class="oui-select__input" name="{{configuration.key}}"
-                                        data-ng-attr-id="{{configuration.key + 'Field'}}"
-                                        data-ng-model="configuration.value"
-                                        data-ng-options="conf for conf in configuration.availableValues">
-                                </select>
-                                <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
-                            </div>
-                        </div>
-                    </div>
-                </form>
-            </div>
+                <oui-button
```

Comment:
Should be replaced with oui-form-actions
Missing type=submit.
submit action should be on the form(ng-submit), not on submit button.
Submit button should be always available, never disabled.

Reply:
CX was ok with disabled if data is not modified, I will see we could use the ```oui-form-actions```

Reply:
Yes, guidelines say it cannot be disabled